Makefile

```
@@ -0,0 +1,15 @@
+.PHONY: deploy-stack remove-stack build-nginx
+
+STACK_NAME ?= cfa_opencti
+COMPOSE_FILE?= docker-compose.yml
+NGINX_TAG?=0.0.1
+
+build-nginx:
```

```suggestion
build:
```

I'm assuming we're not building anything else.